src/OmniSharp.Abstractions/IOmniSharpEnvironment.cs

```diff
@@ -10,5 +10,6 @@ public interface IOmniSharpEnvironment
         string SolutionFilePath { get; }
         string SharedDirectory { get; }
         string[] AdditionalArguments { get; }
+        bool WantTelemetryInfo { get; }
```

Comment:
nit: I know names are hard and this isn't a blocker but maybe `IncludeTelemetryInfo` instead

Reply:
I'm happy to make that change. What about the flag names?  `--include-telemetry-info` and `-it` ?